sceptre/connection_manager.py

```diff
@@ -240,14 +244,15 @@ def call(
         """
         if region is None and profile is None and iam_role is None:
             if stack_name and stack_name in self._stack_keys:
-                region, profile, iam_role = self._stack_keys[stack_name]
+                region, profile, iam_role, duration_seconds = self._stack_keys[stack_name]
             else:
                 region = self.region
                 profile = self.profile
                 iam_role = self.iam_role
+                duration_seconds = self.duration_seconds
 
         if kwargs is None:  # pragma: no cover
             kwargs = {}
 
-        client = self._get_client(service, region, profile, stack_name, iam_role)
+        client = self._get_client(service, region, profile, stack_name, iam_role, duration_seconds)
```

Comment:
I don't think anywhere in the Sceptre Codebase will be invoking this method with `duration_seconds`. I think it would be best to just use the `duration_seconds` instance variable.